auditbeat/include/fields.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 func init() {
-	if err := asset.SetFields("auditbeat", "fields.yml", Asset); err != nil {
+	if err := asset.SetFields("auditbeat", "fields.yml", 1, Asset); err != nil {
```

Comment:
@andrewkroh @cwurm  I had to create this manually. It seems the auditbeat fields.go does not get updated properly.